### PR TITLE
Add guidelines for running breakfast

### LINKED ISSUE #211: Allocate an area for breakfast
Allocate an area for breakfast every morning in Galvanize and add it to the playbook.

Perhaps communicate with Galvanize staff to try and secure a place that can support 14 hackers + leaders and us? That's a lot of people.

### PR DESCRIPTION
**Pull request by zachlatta**
@MaxWofford can you take a look at (and potentially merge) this when you have a chance?

This closes #211 and #212. Instead of communicating with Galvanize directly and trying to get an allocated space, due to time constraints I'm thinking about just having breakfast on the roof and dealing with the inconvenience of it.

/cc @jonleung

### CODE REVIEW

File: cohort_4/playbook/activities.md [hunk 1]
```diff
@@ -43,6 +44,31 @@ arriving.
...
+2. To set up breakfast, the leader will need to:
+  1. Bring the food we have for breakfast from the third floor to the roof of
+     the building. Our food is in the cabinets below the stove in the kitchen
+     past the open working space on the third floor (not the kitchen right next
+     to the stairs).
+  2. Move tables together on the roof and bring chairs together.
+3. Once the leader has set up breakfast, and when it's time to do breakfast,
+   have everyone take the stairs to the roof.
+
+The hackers may be pretty quite during breakfast, especially on the first day,
```
Comment (maxwofford) [inline cohort_4/playbook/activities.md]: quite -> quiet

**Comment by maxwofford** [comment]
@zachlatta I completely agree that we don't have time to deal with allocating space right now. Just a single comment, then it's ready to merge :-)

**Comment by jonleung** [comment]
@zachlatta I sent an email to Kat asking for the 3rd floor space for breakfast as well because I think that the extra time for anyone to go from the 3rd floor to the roof and back again is time that we don't have while actually running the cohort.

Feel free to merge for now an we can change it later?

**Comment by zachlatta** [comment]
Sounds good!

## 

Zach Latta
Executive Director of hackEDU https://hackedu.us
http://zachlatta.com
https://github.com/zachlatta
https://twitter.com/zachlatta

**Pull request merged**